internal/tokens/fftokens/fftokens.go

```diff
@@ -349,11 +349,12 @@ func (ft *FFTokens) handleTokenPoolCreate(ctx context.Context, data fftypes.JSON
 		}
 	}
 
-	txType := poolData.TXType
-	if txType == "" {
+	if poolData != nil {
```

Comment:
Looks like the fix needed here, is to cover the `poolData == nil && poolDataString == ""` case (setting `poolData = &tokenData{}`)